### PR TITLE
feat: Add functions to get/update realm users profile

### PR DESCRIPTION
**Pull request by asyd**
Hello there, 

a small PR to add function to get/put users profiles. Needed to manage users attributes for recent Keycloak versions.

### CODE REVIEW

File: src/keycloak/keycloak_admin.py [hunk 1]
```diff
@@ -2911,6 +2929,26 @@ def update_realm_role(self, role_name: str, payload: dict) -> bytes:
...
+        https://www.keycloak.org/docs-api/26.0.0/rest-api/index.html#UPConfig
+        """
+        params_path = {"realm-name": self.connection.realm_name}
+        data_raw = self.connection.raw_put(
+            urls_patterns.URL_ADMIN_REALM_USER_PROFILE.format(**params_path),
+            data=json.dumps(payload),
+        )
+        return raise_error_from_response(
+            data_raw,
+            KeycloakDeleteError,
```
Comment (ryshoooo) [inline src/keycloak/keycloak_admin.py]: Shouldn't this be `KeycloakPutError`?

File: src/keycloak/keycloak_admin.py [hunk 2]
```diff
@@ -5490,6 +5528,26 @@ async def a_update_realm(self, realm_name: str, payload: dict) -> dict:
...
+        https://www.keycloak.org/docs-api/26.0.0/rest-api/index.html#UPConfig
+        """
+        params_path = {"realm-name": self.connection.realm_name}
+        data_raw = await self.connection.a_raw_put(
+            urls_patterns.URL_ADMIN_REALM_USER_PROFILE.format(**params_path),
+            data=json.dumps(payload),
+        )
+        return raise_error_from_response(
+            data_raw,
+            KeycloakDeleteError,
```
Comment (ryshoooo) [inline src/keycloak/keycloak_admin.py]: Same here

File: tests/test_keycloak_admin.py [hunk 3]
```diff
@@ -204,6 +204,15 @@ def test_realms(admin: KeycloakAdmin) -> None:
     assert "master" in realm_names, realm_names
     assert "test" in realm_names, realm_names
 
+    # Get and update users profile
+    user_profile = admin.get_realm_users_profile()
+    assert "attributes" in user_profile
+    assert "groups" in user_profile
+
+    res = admin.update_realm_users_profile(user_profile)
```
Comment (ryshoooo) [inline tests/test_keycloak_admin.py]: Could we also test updating some attributes/groups? Just to be sure that we're not only hitting an edge case here as the profile is unchanged.

**Review by ryshoooo — COMMENTED** [review]
Thanks a lot @asyd ! Overall looks good, just left a few comments :)

File: tests/test_keycloak_admin.py (hunk 3 above)
Comment (asyd) [reply tests/test_keycloak_admin.py]: Sure!

**Comment by asyd** [comment]
I skipped test for versions < 24. I guess the API was not stable before (no easy to find KC changelog for versions around 22)

**Review by ryshoooo — APPROVED** [review]
Thanks a lot! LGTM
